src/main/java/com/seailz/discordjv/model/guild/Guild.java

```diff
@@ -834,6 +836,21 @@ public CreateGuildChannelAction createChannel(String name, ChannelType type) {
         return new CreateGuildChannelAction(name, type, this, discordJv);
     }
 
+    /**
+     * Deletes the Guild. The app must own the Guild in order to delete it.
+     * This cannot be undone.
+     */
+    public void delete() {
+        DiscordResponse res = new DiscordRequest(
+                new JSONObject(),
+                new HashMap<>(),
+                URLS.GET.GUILDS.GET_GUILD.replace("{guild.id}", id),
+                discordJv,
+                URLS.BASE_URL,
```

Comment:
This should be the URL of the request with no changes, so for example if you had
`URLS.FOO.replaceAll("hello", "goodbye")`,
You'd set the base URL to
`URLS.FOO`